From 17fbe9757c3c0685c0db88ba5921e6bfb6e61cda Mon Sep 17 00:00:00 2001
Message-ID: <17fbe9757c3c0685c0db88ba5921e6bfb6e61cda.1716242564.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1716242563.git.sharlatanus@gmail.com>
References: <cover.1716242563.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 15 May 2024 13:46:33 +0100
Subject: [PATCH 13/26] gnu: python-crds: Update to 11.17.22.

* gnu/packages/astronomy.scm (python-crds): Update to 11.17.22. Adjust
indentation.
[native-inputs]: Remove python-flake8, python-ipython,
python-lockfile, python-mock, python-nose, python-pylint, and
python-pytest.

Change-Id: Idbbece6544755753dce2046292d2552924992959
---
 gnu/packages/astronomy.scm | 45 +++++++++++++++++---------------------
 1 file changed, 20 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index c6baece0f7..38b878ab25 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2150,13 +2150,13 @@ (define-public python-cmyt
 (define-public python-crds
   (package
     (name "python-crds")
-    (version "11.17.20")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "crds" version))
-              (sha256
-               (base32
-                "0480jjklv2p60fp892dr4qglwv44k86hx29y8q1c967l9k53wc9f"))))
+    (version "11.17.22")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "crds" version))
+       (sha256
+        (base32 "1i2wh7cgyn7r0fnn7ikfsdqjrcpg46xsk619mhdqywk5yjcxlz9f"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -2164,24 +2164,19 @@ (define-public python-crds
       ;; additional test data. See:
       ;; https://github.com/spacetelescope/crds/blob/master/setup_test_cache
       #:tests? #f))
-    (propagated-inputs (list python-asdf
-                             python-astropy
-                             python-boto3
-                             python-filelock
-                             python-numpy
-                             python-parsley
-                             python-pysynphot
-                             python-roman-datamodels
-                             python-stsynphot
-                             python-requests))
-    (native-inputs (list python-flake8
-                         python-ipython
-                         python-lockfile
-                         python-mock
-                         python-nose
-                         python-pylint
-                         python-pytest
-                         python-setuptools-scm))
+    (propagated-inputs
+     (list python-asdf
+           python-astropy
+           python-boto3
+           python-filelock
+           python-numpy
+           python-parsley
+           python-pysynphot
+           python-roman-datamodels
+           python-stsynphot
+           python-requests))
+    (native-inputs
+     (list python-setuptools-scm))
     (home-page "https://hst-crds.stsci.edu")
     (synopsis "Calibration Reference Data System for HST and JWST")
     (description
-- 
2.41.0

